src/log_capture.jl

```diff
@@ -241,35 +241,44 @@ function _print_test_errors(report_iob, ts::DefaultTestSet, worker_info)
     return nothing
 end
 
-# Marks the start of each test item
-function log_testitem_start(ti::TestItem, ntestitems=0)
-    io = IOContext(IOBuffer(), :color => get(DEFAULT_STDOUT[], :color, false)::Bool)
+function print_state(io, state, ti, ntestitems)
     interactive = parse(Bool, get(ENV, "RETESTITEMS_INTERACTIVE", string(Base.isinteractive())))
     print(io, format(now(), "HH:MM:SS | "))
     !interactive && print(io, _mem_watermark())
-    printstyled(io, "START"; bold=true)
+    printstyled(io, uppercase(state); bold=true)
```

Comment:
This refactor seems to break the alignment we had before. I.e.:
```
  Worker 60931:  11:33:49 | SKIPPED (2/3) test item "skip expr true" at test/testfiles/_skip_tests.jl:21
  Worker 60932:  11:33:49 | START (1/3) test item "skip expr false, 2 pass" at test/testfiles/_skip_tests.jl:16
  Worker 60932:  11:33:49 | DONE (1/3) test item "skip expr false, 2 pass" <0.1 secs (91.9% compile), 45.88 K allocs (3.242 MB)
```
I think it would be preferable to have:
```
  Worker 60931:  11:33:49 | SKIP  (2/3) test item "skip expr true" at test/testfiles/_skip_tests.jl:21
  Worker 60932:  11:33:49 | START (1/3) test item "skip expr false, 2 pass" at test/testfiles/_skip_tests.jl:16
  Worker 60932:  11:33:49 | DONE  (1/3) test item "skip expr false, 2 pass" <0.1 secs (91.9% compile), 45.88 K allocs (3.242 MB)
```

Reply:
thanks, fixed with a test